.github/workflows/deploy-synopsys-extension.yml

```diff
@@ -0,0 +1,88 @@
+name: deploy-synopsys-extension
+
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy-synopsys-extension:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set Node.js 20.x
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+
+      - name: versioning
+        id: version-update
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUBTOKEN }}
+        run: |
+          vss_extension=$(cat vss-extension.json)
```

Comment:
Can we combine first two lines 

```
vss_extension=$(cat vss-extension.json)
          extension_name=$(echo $vss_extension | jq -r '.name' ) 
```
TO extension_name=$(jq -r '.name' < vss-extension.json)

Reply:
changes are done